packages/shopify-app-session-storage-postgresql/src/postgresql.ts

```diff
@@ -151,7 +151,7 @@ export class PostgreSQLSessionStorage implements SessionStorage {
     const hasSessionTable = await this.hasSessionTable();
     if (!hasSessionTable) {
       const query = `
-        CREATE TABLE ${this.options.sessionTableName} (
+        CREATE TABLE IF NOT EXISTS ${this.options.sessionTableName} (
```

Comment:
Can you add a comment above to explain why using `pg_catalog.pg_tables` doesn't work anymore from `postsgre 15` and link [this](https://www.postgresql.org/docs/release/15.0/).

Reply:
Ahhh hasSessionTable should totally be working but it looks like pg_tables was removed from pg_catalog, is this correct?

Reply:
Also, I went ahead and implemented the requested changes. I ran the test suite locally and passed but looks like I need a maintainer to approve running workflows since its my first contribution to the repo.